packages/shared/src/server/repositories/eventLog.ts

```diff
@@ -258,3 +259,35 @@ export const deleteEventLogByProjectIdBeforeDate = async (
     },
   });
 };
+
+export const getEventLogOrderedByTime = async (
```

Comment:
It may make sense to leave a comment that this is supposed to be used for the migration.